### DISCUSSION TITLE
How to color the streamlines as per directions?

### DISCUSSION THREAD

**Discussion by Sunita2023** [dipy/dipy#3362] [Tractography / Tractometry]
Dear Members,

We want to show the streamlines with standard color codes (RGB) which reflects directions. However, fury shows deviation from the actual colors. For the same bundle DSI studio shows correct colors as per directions while fury shows different colors. Please have a look at the attached files.  

This image is shown in Fury. 
![cc_fury_image](https://github.com/user-attachments/assets/02139622-f290-47ee-9674-6e59b61ac78f)

This image is shown in DSI Studio. 
![cc_dsistudiol](https://github.com/user-attachments/assets/5701f636-49ee-4d59-a40d-05ff5801e646)

Is there any built-in function to show the correct colors as per directions?

I would highly appreciate DIPY developers for their incredible supports to the users.   

Best regards,
Sunita

**Comment by Sunita2023** [comment]
Dear Jong,

Could you please suggest if there is any built-in function to color the streamlines correctly. 
Thank you so much. 

Best regards,
Sunita

**Reply by pjsjongsung** [reply]
Hi, sorry for the late response. I believe `fury.actor.line` does use a orientation map when displaying the colors. But if you could provide your code that was used to visualize the image, it would be easier to understand what is necessary. Also, @skoudoro and @guaje might be better equipped in answering this question.

**Reply by skoudoro** [reply]
Can you share this part of your code @Sunita2023?

I think the DTI tutorial shows an example concerning color per directions: https://docs.dipy.org/stable/examples_built/reconstruction/reconst_dti.html#sphx-glr-examples-built-reconstruction-reconst-dti-py
